main.tf

```diff
@@ -52,7 +52,25 @@ resource "random_string" "cf_key" {
   special = false
 }
 
+# https://github.com/hashicorp/terraform-provider-aws/issues/30951#issuecomment-1523376909
+resource "aws_s3_bucket_public_access_block" "logging_bucket_public_enabled" {
```

Comment:
Why would the logging bucket need to be public?

Reply:
It needs to allow ACLs to let CloudFront write access logs to it.
https://github.com/byu-oit/hw-static-site/actions/runs/4835316093/jobs/8617496935#step:10:32